### PR TITLE
Add code_of_conduct

### PR DESCRIPTION
**Pull request by hollenberry**
## Overview
**TL;DR**
In an effort to make our repository more beginner friendly, this PR aims to update the CONTRIBUTING.md with more details. 

Related to: https://github.com/github/training-kit/issues/415

### Next Steps
- [ ] Add more content to outline/review outline
- [ ] Add links to existing resources
- [ ] Populate content

Some thoughts:
- I imagine that a lot of this content will be shareable or similar to the Open Source content that will be produced. As far as timelines go, I'm hoping these launch at a similar time (February). 
- [Atom's CONTRIBUTING.md](https://github.com/atom/atom/blob/master/CONTRIBUTING.md) is where I found the initial resources.
- I ❤️  the Q+A style for common questions in the [Codebuddies CONTRIBUTING.md](https://github.com/codebuddiesdotorg/codebuddies/blob/master/contributing.md#helpful-reminders) . I think this might be a nice resource to contribute to based on Gitter chats and items we might see pop up in issues. 
- I ❤️ the format of the [Hoodie CONTRIBUTING.md](https://github.com/hoodiehq/hoodie/blob/master/CONTRIBUTING.md) . I'd love to do something similar when we provide instructions, which also echoes how we've produced content within some pages of our courses. 

cc: @github/services-training ✨

### CODE REVIEW

**Comment by brianamarie** [comment]
@hollenberry ✨ ✨ ✨  What can I do to help?

**Comment by hollenberry** [comment]
@brianamarie Whatever you're interested in taking on! If you want to add or remove sections and start populating content, I welcome anyone to update directly. 😄 

There might be overlap or overwriting later, but if there's anything that you're excited to tackle, feel free!

**Comment by crichID** [comment]
@github/services-training based on the work happening in #571, I have done the following on this branch:

- Removed the stub of the CONTRIBUTING guide since it is being worked on in #571 
- Removed the Contributor code of conduct from the community.md page since it will now live in a separate doc
- Applied some minor edits to the CODE_OF_CONDUCT to fix copy/paste issues

I would like to go ahead and :ship: this so I can link it from the CONTRIBUTING guide in #571. Can I please get a couple sets of :eyes:, ✏️ s, and :+1:s?

**Comment by crichID** [comment]
I think the only thing remaining here is to double check that there are no other links to the code of conduct in training-kit that are expecting it to be in the community.md. @hollenberry in case you have time :wink:
